models/mail.go

```diff
@@ -47,7 +47,8 @@ func SendTestMail(email string) error {
 // SendUserMail sends a mail to the user
 func SendUserMail(c *macaron.Context, u *User, tpl base.TplName, code, subject, info string) {
 	data := map[string]interface{}{
-		"Username":          u.DisplayName(),
+		"DisplayName":       u.DisplayName(),
+		"Username":          u.DisplayUsername(),
```

Comment:
This can just be `u.Name`, no need for a func.

Reply:
oh yes duh 🙈 fixed